app/assets/stylesheets/app.scss

```diff
@@ -92,12 +92,12 @@ p {
 
 /* forms */
 
-input, textarea, select, .uneditable-input {
-  border: 1px solid #bbb;
-  width: 100%;
-  margin-bottom: 15px;
-  @include box_sizing;
-}
+// input, textarea, select, .uneditable-input {
+//   border: 1px solid #bbb;
+//   width: 100%;
+//   margin-bottom: 15px;
+//   @include box_sizing;
+// }
```

Comment:
Usually you don't want to keep unused code around since it will pollute your code searches and add clutter.